src/test/denialofservice_tests.cpp

```diff
@@ -346,12 +346,6 @@ BOOST_AUTO_TEST_CASE(DoS_bantime)
     }
     BOOST_CHECK(banman->IsBanned(addr));
 
```

Comment:
```suggestion
     BOOST_CHECK_EQUAL(1, banman->IsBannedLevel(addr));
```

nit: Could add a test for `IsBannedLevel`?